tests/kernel/workq/critical/src/main.c

```diff
@@ -166,7 +166,7 @@ void regression_thread(void *arg1, void *arg2, void *arg3)
 		      "Unexpected value for <critical_var>");
 
 	TC_PRINT("Enable timeslicing at %u\n", k_uptime_get_32());
-	k_sched_time_slice_set(10, 10);
+	k_sched_time_slice_set(20, 10);
```

Comment:
Minor: shouldn't this be pulled from .conf?

Reply:
Yes, but it wasn't before so it isn't now. :)  My personal feeling about having ugly code in the regression suite is that it's far more important that it stay there and run than experience churn that might hide bugs we'd otherwise detect.  Zephyr is expected to run bad code correctly too.

(Not like I'd reject a patch that cleaned up hardcoded timeslices or anything, just that it isn't my personal inclination to try to "clean up" tests at the same moment I'm trying to make them pass.)